apps/api/src/app/auth/e2e/user-registration.e2e.ts

```diff
@@ -96,7 +96,7 @@ describe('User registration - /auth/register (POST)', async () => {
       email: 'Testy.test@gmail.com',
       firstName: 'Test',
       lastName: 'User',
-      password: '123456789',
+      password: '123@Qwerty',
```

Comment:
Do we have a test today to validate the password strength logic? Maybe we can create a small case for it here 🙏

Reply:
Couldn't find any exisitng tests,
will add them